repository/l.json

```diff
@@ -769,17 +769,21 @@
 		},
 		{
 			"name": "LESS",
-			"details": "https://github.com/danro/LESS-sublime",
+			"details": "https://github.com/SublimeText/LESS",
 			"donate": "https://paypal.me/koenlageveen",
```

Comment:
Let's remove the donation bit. I haven't received anything in a while, and it's no longer my work.